pkg/k8s/k8s_pod_probes.go

```diff
@@ -0,0 +1,66 @@
+package k8s
+
+import "fmt"
+
+// To keep the structs minimal, everything that is not needed right now is commented out
+// If you need it, uncomment it and add the needed fields
+// This way we can keep the structs minimal and add more without a breaking change
+
+// Probe is a structure to hold the probes for an instance
+type Probe struct {
+	//Exec                          *ExecAction
+	HTTPGet             *HTTPGetAction   // Only one action is allowed, deployment will fail otherwise
+	TCPSocket           *TCPSocketAction // Only one action is allowed, deployment will fail otherwise
+	InitialDelaySeconds int32            // Optional
+	//TerminationGracePeriodSeconds int64
+	//PeriodSeconds                 int32
+	//TimeoutSeconds                int32
+	//FailureThreshold              int32
+	//SuccessThreshold              int32
+	//GRPC                          *GRPCAction
```

Comment:
Can you elaborate on why we need to comment this out?

Reply:
https://github.com/celestiaorg/knuu/pull/127/files#diff-97b6d1c19f51934d9900ee900ffd698b15a1750ea95088d1f30ff73d9a98fb5dR5-R7

Reply:
This is not a good explanation and the way to go tbh. I'd propose you start with Interfacing to achieve duck typing and leave room for future improvements with backwards compat. 

Rather than off-setting the choice to the user to uncomment and do the implementation